docs/pact-reference.md

```diff
@@ -1354,8 +1355,8 @@ runtime error.
 
 ### References {#references}
 
-References are two atoms joined by a dot `.` to directly resolve to module
-definitions.
+References are two atoms joined by a dot `.` that directly resolve to definitions found
```

Comment:
Hmm, maybe we should call these "Qualified names", does this "references" language show up anywhere else?

Reply:
Once we have namespaces, we will be using more than one qualifier, so perhaps consider length-independent language :)

Reply:
I distinctly recall a line that says "You can use reference syntax in order to foobarbaz. See the References section for more info".